scipy/interpolate/tests/test_bsplines.py

```diff
@@ -635,7 +634,6 @@ def test_read_only(self):
         b = BSpline(t=t, c=c, k=0)
         xp_assert_close(b(xx), np.ones_like(xx) * 3.0)
 
-    @pytest.mark.thread_unsafe
```

Comment:
I think this marker was put in place to prevent threads from spawning more threads, given that the test already runs in parallel